api/src/cli/utils/create-env/env-stub.liquid

```diff
@@ -1,60 +1,331 @@
+# These values set variables which modify core settings of Directus. The variables can be stored
+# directly in your environment (which means the variables are accessible from the operating system)
+#
+# It is more performant to set these values in your environment directly rather than in this file
+# but can be left here whilst developing in case you want to change them. If you set all the 
+# variables directus needs directly in your environment, there is no need for this file.
+#
+# To set permanent environment variables, create an .sh file in the "/etc/profile.d" folder and 
+# add one of the lines below prefixed with "export ". Once done, log out and back in to set it.  
+#
+# e.g. To set the port that directus listens on, create a file "/etc/profile.d/directus.sh". (This 
+# name is not fixed and could be "/etc/profile.d/myprojectname.sh".) Then add the line:
+# export PORT=8055
+#
+# Log out of shell and back in and the variable will be set. To remove the variable, remove the
+# line and log out and back in again.
+#
+# 
+# The below are not all possible options, for more, see 
+# https://docs.directus.io/configuration/config-options/
+# 
+####################################################################################################
+####################################################################################################
+
+### General
 
+# IP or host the API listens on ["0.0.0.0"]
 HOST="0.0.0.0"
+
+# The port Directus will run on [8055]
 PORT=8055
+
+# The URL where your API can be reached on the web. It is also used for things like OAuth redirects, 
+# forgot-password emails, and logos that needs to be publicly available on the internet. ["/"]
 PUBLIC_URL="/"
+# PUBLIC_URL="http://localhost:8055"
+
+# What level of detail to log. [info]
+# "fatal", "error", "warn", "info", "debug", "trace", "silent"
+# LOG_LEVEL="info"
+
+# Render the logs human readable (pretty) or as JSON (raw), [pretty]
+# "pretty", "raw"
+# LOG_STYLE="pretty"
+
+# Controls the maximum request body size. Accepts number of bytes, or human readable string ["100kb"]
+# MAX_PAYLOAD_SIZE="100kb"
+
+# Where to redirect to when navigating to /. Accepts a relative path, absolute URL, or false to disable ["./admin"]
+# ROOT_REDIRECT="./admin"
+
+# Whether or not to serve the Admin App under /admin. [true]
+# SERVE_APP=true
+
+# Whether or not to enable GraphQL Introspection [true]
+# GRAPHQL_INTROSPECTION=true
 
 ####################################################################################################
-## Database
+### Database
+
+# All DB_* environment variables are passed to the connection configuration of a Knex instance. 
+# Based on your project's needs, you can extend the DB_* environment variables with any config 
+# you need to pass to the database instance.
 
 {{ database }}
 
+
+# These match the databases defined in the docker-compose file in the root of this repo
+
+## Postgres
+# DB_CLIENT="pg"
+# DB_HOST="localhost"
+# DB_PORT=5100
```

Comment:
Lets use the default ports for the database providers. The ports shown here are the ones that we use in the debugging docker-compose file, but those are arbitrary numbers that don't match "regular" use of these vendors.

Reply:
done